### PR TITLE
dbeaver/dbeaver#16323 change icon for the 'Open in external editor' action

### LINKED ISSUE #16323: Change icon for the 'Open in external editor' action
This button is located on the value panel. Currently, it looks like three horizontal dots. It has two drawbacks:

* It's counterintuitive.
* It's almost invisible, especially on dark themes.

Screenshot from 2022-04-26 22-45-07

### PR DESCRIPTION
**Pull request by flambchap**
Screenshot from 2022-04-26 22-38-17

### CODE REVIEW

File: plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/ContentValueManager.java [hunk 1]
```diff
@@ -82,7 +82,7 @@ public void run() {
                 });
             }
             // Logo can be changed
-            manager.add(new Action("Open in external editor", DBeaverIcons.getImageDescriptor(UIIcon.DOTS_BUTTON)) {
+            manager.add(new Action("Open in external editor", DBeaverIcons.getImageDescriptor(UIIcon.FILE)) {
```
Comment (LonwoLonwo) [inline plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/ContentValueManager.java:84]: "Open in external editor" can be localized :smiling_imp:

Comment (LonwoLonwo) [reply plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/ContentValueManager.java:84]: Also, what about "// Logo can be changed" comment then? You changed the logo.

Comment (flambchap) [reply plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/data/managers/ContentValueManager.java:84]: Good point

**Review by ShadelessFox — APPROVED** [review]

**Comment by Matvey16** [comment]
Verified

**Pull request merged**